edgecast/logging/logger_test.go

```diff
@@ -0,0 +1,15 @@
+package logging
+
+import (
+	"testing"
+)
+
+func TestSample(t *testing.T) {
+
+	got := 5
+	want := 10
+
+	if got != want {
```

Comment:
Not sure what's the purpose of this test.

Reply:
I think she just needs a failed test example to test the PR functionality